integrations.go

```diff
@@ -158,6 +158,51 @@ func (client *Client) DeleteIntegrationSlack() error {
 	return client.doJsonRequest("DELETE", "/v1/integration/slack", nil, nil)
 }
 
+/*
+	Webhook Integration
+*/
+
+// IntegrationWebhookRequest defines the structure of the a webhook request
+type IntegrationWebhookRequest struct {
+	Webhooks []Webhook `json:"hooks"`
+}
+
+// Webhook defines the structure of the a webhook
+type Webhook struct {
+	Name             *string `json:"name"`
+	URL              *string `json:"url"`
+	UseCustomPayload *string `json:"use_custom_payload"`
+	CustomPayload    *string `json:"custom_payload"`
+	EncodeAsForm     *string `json:"encode_as_form"`
+	Headers          *string `json:"headers"`
+}
+
+// CreateIntegrationWebhook creates a new Webhook Integrations.
+func (client *Client) CreateIntegrationWebhook(webhookIntegration *IntegrationWebhookRequest) error {
+	return client.doJsonRequest("POST", "/v1/integration/webhooks", webhookIntegration, nil)
+}
+
+// UpdateIntegrationWebhook updates the Webhook Integration.
+// This will add to the existing values with the new values, and not _delete_ it
```

Comment:
I think this comment is a little bit misleading. Using the PUT method *replaces* the current configuration with the configuration sent in the request. I'd suggest something like:

```
// UpdateIntegrationWebhook updates the Webhook Integration.
// It replaces the existing configuration with the configuration sent in this request.
```

Reply:
Done!